int64 chargeTimeout
string chargeCommand
----
string chargeResult
